internal/driver/driver.go

```diff
@@ -351,6 +418,37 @@ func (d *Driver) HandleWriteCommands(deviceName string, protocols map[string]mod
 			}
 		case VideoStopStreaming:
 			device.StopStreaming()
+		case VideoSetFrameRate:
+			fpsParam, edgexErr := params[i].ObjectValue()
+			if edgexErr != nil {
+				return errors.NewCommonEdgeXWrapper(edgexErr)
+			}
+			fpsValueDenominator, ok := fpsParam.(map[string]interface{})[FpsValueDenominator]
```

Comment:
the naming of this in the API input can be a little confusing. technically these are the "interval" values which are an inverse of the actual fps. its actually spf (sunblock anyone)? seconds per frame. I think maybe we name it `FpsInvervalDenominator`? Either way we probably need to document it.

Reply:
Yeah that was a problem during development, the fact that internally it's handled opposite of how we usually think about frame rates.